hirosystems/stacks-blockchain-api/values.yaml

```
@@ -74,10 +74,29 @@ apiWriter:
     # dataArchiveUrl: https://storage.googleapis.com/hirosystems-archive/mainnet/api/mainnet-blockchain-api-5.0.0-latest.tar.gz
     # Replay events from the API's TSV file into the Postgres database
     # CAUTION: THIS WILL WIPE YOUR CURRENT DATABASE
+
+    # Note: both exportEvents.enabled and exportEvents.compressEventsData should be set to 'true' in most common situations
+    # followed by replayEvents also set to 'true'
```

Clarifying for users unaware of this change
```suggestion
    # Note: exportEvents.compressEventsData should be set to 'true' when you intend to replayEvents immediately after
```